packages/ctl/ctl.js

```diff
@@ -903,6 +903,29 @@ userCommands.add(new libCommand.Command({
 	},
 }));
 
+userCommands.add(new libCommand.Command({
+	definition: ["admin-bulk-import <file>", "Bulk input from file users to be admin", (yargs) => {
+		yargs.positional("file", { describe: "File to import from", type: "string" });
+		yargs.options({
+			"create": { describe: "Create user if it does not exist", nargs: 0, type: "boolean", default: false },
+		});
+	}],
+	handler: async function(args, control) {
+		let data = await fs.readFile(args.file, "utf8");
+		let names = data.split("\n");
+		for (let name in names) {
+			if (name in names) {
```

Comment:
What is the point of this though?

Reply:
Not sure myself. eslint has spit out errors in the past if it’s not there, so I preemptively added it. Unfortunately I can’t test if it’s needed right now, but if it’s not, i’d be happy to remove it.

Reply:
`for (... in ...)` loops are an error more often than not, you shouldn't use them.

Reply:
OK. I’ll replace it next commit.